flow/flow-fetcher/starport/flow_lock_event.go

```diff
@@ -0,0 +1,34 @@
+package starport
```

Comment:
Generally throughout this file should we be using checked access with err return values? What happens when you do `evt.Fields[0]` but `len(evt.Fields) == 0` ? Is that a go panic at that point? Same goes for unchecked casts in this file